tests/fixtures/issues/issue-317.php

```diff
@@ -1657,7 +1657,7 @@
             'MajorVer' => '9',
             'MinorVer' => '4',
             'Platform' => 'Android',
-            'Platform_Version' => '4.0',
```

Comment:
The changes in these test files were because the UA that these are testing are of this variety:

>UCWEB/2.0 (Linux; U; Opera Mini/* #DEVICE#)

Where there is no actual platform indicator. It seems that the android version used was based on the phone's capabilities.  Many of the devices that were detected this way support multiple versions of android, so I moved those to just a generic Android platform (which is why this changed to `unknown`). In other cases, the source I had for the phone showed a different version than what was defined.

In general it's probably better to move all of these to a generic "Android" platform with no version, since there's really no way to tell which version of android is installed on the phone from just the UA.

I can make this change if that's the way we want to go with that.